schemas/CMakePresets-v6-schema.json

```diff
@@ -0,0 +1,1579 @@
+                            }
+                        },
+                        "additionalProperties": false
+                    },
+                    "packageName": {
+                        "type": "string",
+                        "description": "An optional string representing the package name. Equivalent to passing -P on the command line."
+                    },
+                    "packageVersion": {
+                        "type": "string",
+                        "description": "An optional string representing the package version. Equivalent to passing -R on the command line."
+                    },
+                    "packageDirectory": {
+                        "type": "string",
+                        "description": "An optional string representing the directory in which to place the package. Equivalent to passing -B on the command line."
+                    },
+                    "condition": {
+                        "$ref": "#/definitions/topCondition"
+                    }
+                },
+                "required": [
+                    "name"
+                ],
+                "additionalProperties": false
+            }
+        },
+
+        "workflowPresets": {
+            "type": "array",
+            "description": "An optional array of workflow preset objects. Available in version 6 and higher.",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "name": {
+                        "type": "string",
+                        "description": "A required string representing the machine-friendly name of the preset. This identifier is used in the '--workflow --preset' CMake argument. There must not be two workflow presets in the union of CMakePresets.json and CMakeUserPresets.json in the same directory with the same name, but a workflow preset may have the same name as a configure, build, test or package preset.",
+                        "minLength": 1
+                    },
+                    "vendor": {
+                        "type": "object",
+                        "description": "An optional map containing vendor-specific information. CMake does not interpret the contents of this field except to verify that it is a map if it does exist. However, it should follow the same conventions as the root-level vendor field.",
+                        "properties": {}
+                    },
+                    "displayName": {
+                        "type": "string",
+                        "description": "An optional string with a human-friendly name of the preset."
+                    },
+                    "description": {
+                        "type": "string",
+                        "description": "An optional string with a human-friendly description of the preset."
+                    },
+                    "steps": {
+                        "type": "array",
```

Comment:
This may be where you need to add `additionalProperties` as false. 

Also, if you want a schema to search through for comparison. https://cmake.org/cmake/help/latest/_downloads/3e2d73bff478d88a7de0de736ba5e361/schema.json

Reply:
Thanks for spotting this and for the link. I wasn't aware I have such schema at my disposal for inspiration.